packs/orion/actions/get_discovery_progress.py

```diff
@@ -0,0 +1,57 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import xml.etree.ElementTree as ET
+
+from lib.actions import OrionBaseAction
+# from lib.utils import discovery_status_to_text
+
+
+class GetDiscoveryProgress(OrionBaseAction):
+    def run(self, profileId, platform):
```

Comment:
We tend to use underscore separated variable names, but if you are just replicating origin parameter names I'm fine with using camel case...